.eslintrc.json

```diff
@@ -0,0 +1,115 @@
+{
```

Comment:
Could we maintain the other file and add only the rules what we need?

Reply:
This one I deleted it just for me to create from 0 everything and learn because actually, it was configured :) but I can recover it, you tell me

Reply:
Yes you can modify the .cjs and add only the rules that are more important. I saw in .json file a lot of rules, and maybe some on them are not so important